### PR TITLE
add healthcheck cmd to probe live and ready

### PR DESCRIPTION
**Pull request by A-Zee029**
#### What type of this PR
- Performance

#### Which issue(s) this PR fixes:
#1153

### CODE REVIEW

File: dist/images/install.sh [hunk 1]
```diff
@@ -1921,15 +1921,13 @@ spec:
           readinessProbe:
             exec:
               command:
-                - bash
-                - /kube-ovn/kube-ovn-controller-healthcheck.sh
+                - ./kube-ovn-controller-healthcheck
```
Comment (zhangzujian) [inline dist/images/install.sh:1924]: We should use the absolute path `/kube-ovn/kube-ovn-controller-healthcheck` instead.

File: cmd/controller_health_check/controller_health_check.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	"strings"
+	"time"
+)
+
+func CmdMain() {
+	output, err := exec.Command("cat", "/var/run/ovn/ovn-nbctl.pid").CombinedOutput()
+	if err != nil {
+		klog.Errorf("failed to get ovn-nbctl daemon pid, %s", err)
+		os.Exit(1)
+	}
```
Comment (zhangzujian) [inline cmd/controller_health_check/controller_health_check.go:19]: Read the file content using `os.ReadFile`.

File: cmd/controller_health_check/controller_health_check.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	"net"
+	"os"
+	"strings"
+	"time"
+)
+
+func CmdMain() {
+	content, err := os.ReadFile("/var/run/ovn/ovn-nbctl.pid")
+	if err != nil {
+		klog.Errorf("failed to get ovn-nbctl daemon pid, %s", err)
```
Comment (oilbeater) [inline cmd/controller_health_check/controller_health_check.go:16]: it's better to use `klog.Fatalf` to mereg the errorf and exit

Comment (oilbeater) [inline cmd/controller_health_check/controller_health_check.go:16]: it's better to use `klog.Fatalf` to mereg the errorf and exit

**Review by oilbeater — CHANGES_REQUESTED** [review]

**Comment by oilbeater** [comment]
Please squash the commits into one commit

**Comment by zhangzujian** [comment]
`kube-ovn-controller-healthcheck.sh` should be removed.

**Review by oilbeater — APPROVED** [review]

**Comment by oilbeater** [comment]
Thanks! @A-Zee029

**Pull request merged**